### PR TITLE
Added tweak from PyPi 

### PR DESCRIPTION
**Pull request by pinin4fjords**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/tweak/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - tweak
+
+about:
+  home: https://github.com/kislyuk/tweak
+  license: Apache Software
```
Comment (chrisburr) [inline recipes/tweak/meta.yaml]: This should follow the SPDX names (please check I've guessed the right version):
```suggestion
  license: Apache-2.0
```

Comment (pinin4fjords) [reply recipes/tweak/meta.yaml]: Thanks @chrisburr, I'm new to conda-forge and this was just what was generated in the skeleton. I've checked and you're right (though I'm not the package author).

**Comment by pinin4fjords** [comment]
@chrisburr do I need to do anything to trigger the package to be built and made available? Docs don't suggest so, but I'm not seeing a lot going on at https://github.com/conda-forge/tweak-feedstock

**Comment by chrisburr** [comment]
Normally not but there was an error in another recipe which prevented new feedstocks being created. I've fixed it now so it should appear soon.

**Pull request merged**